formatting.css

```diff
@@ -0,0 +1,231 @@
+body, html{
+    margin: 0%;
+    padding: 0%;
+}
+.header-banner{

```

Comment:
make sure to leave a blank line between css selectors and make sure you have spaces before curly brace;
```
h1 {
  color: black;
}

h2 {
  color: blue;
}
```